### ISSUE TITLE
After update from 0.32.0, percentage digits after comma is too high on pizza

### ISSUE THREAD

**Issue by raphamotta** [metabase/metabase#10385]
After update from 0.32 to 0.32.9, percentage digits after comma is too high on pizza. See:

![Captura de tela de 2019-07-20 12-47-03](https://user-images.githubusercontent.com/39523113/61580833-86146c80-aaec-11e9-9abf-2820d3f72d43.png)

Before was only 2 digits after comma. Is there a setting to select the maximum number of digits after comma?

**Comment by flamber** [comment]
Hi @raphamotta 
I cannot reproduce the issue 0.32.9. It just shows 2 decimals. Have you tried to "Reset to defaults" in the visualization settings :gear:, just to make sure it's not a bad setting somewhere?
There was a fix in 0.32.9, which solved some issues with bad rounding/decimals - #10078 - so maybe you're seeing a side-effect from that.
![image](https://user-images.githubusercontent.com/1447303/61581779-c9b4a980-aadf-11e9-8efb-6d1f2d1eb9bf.png)
Using the following data:
```SQL
SELECT 'Neurologia' AS "cat", 31003 AS "num" UNION ALL
SELECT 'Ortopedia' AS "cat", 27912 AS "num" UNION ALL
SELECT 'Cardiologia' AS "cat", 22522 AS "num" UNION ALL
SELECT 'Reumatologia' AS "cat", 14394 AS "num" UNION ALL
SELECT 'Endocrinologia' AS "cat", 11902 AS "num" UNION ALL
SELECT 'Urologia' AS "cat", 6942 AS "num" UNION ALL
SELECT 'Proctologia' AS "cat", 5843 AS "num" UNION ALL
SELECT 'Gastroentero' AS "cat", 5081 AS "num" UNION ALL
SELECT 'Ginecologia' AS "cat", 4858 AS "num" UNION ALL
SELECT 'CirurgiaVasc' AS "cat", 3035 AS "num" UNION ALL
SELECT 'Other1' AS "cat", 1784 AS "num" UNION ALL
SELECT 'Other2' AS "cat", 1784 AS "num" UNION ALL
SELECT 'Other3' AS "cat", 1784 AS "num" UNION ALL
SELECT 'Other4' AS "cat", 1784 AS "num"
```

**Comment by raphamotta** [comment]
Ok see he changes. Maybe because the removed line:
`maximumSignificantDigits: 3,`
on [397ff0c](https://github.com/metabase/metabase/commit/397ff0c67e37fdb823d60118f890255003cda5d1)

You can reproduce using a fresh metabase.jar server on localhost, using the sample database. See:

![Captura de tela de 2019-07-20 17-43-14](https://user-images.githubusercontent.com/39523113/61583983-64c87600-ab16-11e9-9ad1-76aae68c9e5e.png)

Here the sql if you want:

`SELECT parsedatetime(formatdatetime("PUBLIC"."ORDERS"."CREATED_AT", 'yyyyMM'), 'yyyyMM') AS "CREATED_AT", count(*) AS "count"
FROM "PUBLIC"."ORDERS"
GROUP BY parsedatetime(formatdatetime("PUBLIC"."ORDERS"."CREATED_AT", 'yyyyMM'), 'yyyyMM')
ORDER BY parsedatetime(formatdatetime("PUBLIC"."ORDERS"."CREATED_AT", 'yyyyMM'), 'yyyyMM') ASC`

Reset to defaults change nothing. Thanks

**Comment by flamber** [comment]
@raphamotta Great. I can reproduce with Sample Dataset example. Pinging @paulrosenzweig :-)

**Comment by raphamotta** [comment]
I can confirm this issue started with metabase 0.32.8. Just tested 0.32.7 version and it's fine

**Comment by raphamotta** [comment]
Pinging @camsaul since most code between  0.32.7 and 0.32.8 are from him/her. Maybe he/she can help :)

**Comment by flamber** [comment]
@raphamotta Cam doesn't do frontend code, but Paul and Tom are doing that, so I'm guessing you're looking at the wrong place.

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
